test/clojure/nrepl/auth_test.clj

```diff
@@ -0,0 +1,208 @@
+(ns nrepl.auth-test
+  "Authentication system tests.  Should not run tests that expose local
```

Comment:
Authentication or Authorization. ;-)

I find it amusing that even in the scope of the PR we keep mixing those up. ;-)